### PR TITLE
Remove usage of applicationCluster column from WorkspaceCluster

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

We are no longer using the applicationCluster property, this removes it from the TypeORM model entirely.

Migration to drop the column will follow once there's no usage.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Review by geropl — DISMISSED** [review]

File: components/ws-manager-bridge/src/cluster-service-server.ts [hunk 1]
```diff
@@ -345,7 +341,7 @@ function convertToGRPC(ws: WorkspaceClusterWoTLS): ClusterStatus {
     clusterStatus.setScore(ws.score);
     clusterStatus.setMaxScore(ws.maxScore);
     clusterStatus.setGoverned(ws.govern);
-    clusterStatus.setApplicationCluster(ws.applicationCluster);
+    clusterStatus.setApplicationCluster("");
```
Comment (easyCZ) [inline components/ws-manager-bridge/src/cluster-service-server.ts:344]: For now, we're setting it to the empty value. Once this change lands, we can mark the applicationCluster property as reserved, and remove it.

**Comment by easyCZ** [comment]
Resetting @geropl's review as this PR has grown in size (due to dependant cleanup).

**Comment by geropl** [comment]
Requested a preview to test workspace still starting after this change.

**Review by geropl — APPROVED** [review]
Code LGTM, tested and works

**Review by WVerlaek — APPROVED** [review]

**Pull request merged**